src/bosh_azure_cpi/lib/cloud/azure/models/disk_id.rb

```diff
@@ -26,103 +25,77 @@ class DiskId
     #  Parsing id for an existing disk
     #   disk_id = DiskId.parse(id, azure_config)
 
-    VERSION1 = 'v1' # class properties: version (string), id (string), default_resource_group_name (string)
-    VERSION2 = 'v2' # class properties: version (string), id (json)
```

Comment:
Why we don't need the version?

Reply:
I think the 'v1' is just a special case.
do not want to give it a version.

Reply:
and use the @plain_id for the special case.

Reply:
In this case, please add more comment for @plain_id, currently, in the beginning of disk_id.rb and instance_id.rb only have comment for 'v1', 'v2'